src/_guides/libraries/library-tour.md

```diff
@@ -1111,24 +1102,39 @@ In the preceding example, the methods run in the following order:
 2.  `expensiveWork()`
 3.  `lengthyComputation()`
 
+Here is the same code written using await:
+
+<?code-excerpt "lib/library_tour/async/future.dart (then-chain-as-await)"?>
+{% prettify dart %}
+try {
+  final value = await costlyQuery(url);
+  await expensiveWork(value);
+  await lengthyComputation();
+  print('Done!');
+} catch (e) {
+  /* Handle exception... */
+}
+{% endprettify %}
+
 
 #### Waiting for multiple futures
 
 Sometimes your algorithm needs to invoke many asynchronous functions and
-wait for them all to complete before continuing. Use the
-[`Future.wait()`]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-async/Future/wait.html)
+wait for them all to complete before continuing. Use the [Future.wait()][]
 static method to manage multiple Futures and wait for them to complete:
 
-<!-- library-tour/futures/bin/main.dart -->
+<?code-excerpt "lib/library_tour/async/future.dart (wait)" replace="/elideBody;/\/* ... *\//g"?>
 {% prettify dart %}
-Future deleteDone = deleteLotsOfFiles();
-Future copyDone = copyLotsOfFiles();
-Future checksumDone = checksumLotsOfOtherFiles();
+Future deleteLotsOfFiles() async => /* ... */
+Future copyLotsOfFiles() async => /* ... */
+Future checksumLotsOfOtherFiles() async => /* ... */
 
-Future.wait([deleteDone, copyDone, checksumDone])
-    .then((List values) {
-      print('Done with all the long steps');
-    });
+await Future.wait([
+  deleteLotsOfFiles(),
+  copyLotsOfFiles(),
+  checksumLotsOfOtherFiles()
```

Comment:
Nice. 

Style-wise, this looks like a good place for a trailing comma (dartfmt will be forced to always put each element on its own line, and moving elements around is much easier).

Reply:
Yes, good point. Done.